glean-core/src/lib.rs

```diff
@@ -65,6 +65,8 @@ pub struct Configuration {
     pub application_id: String,
     /// The maximum number of events to store before sending a ping containing events.
     pub max_events: Option<usize>,
+    /// Whether Glean should delay persistence of data from metrics with ping lifetime
```

Comment:
```suggestion
    /// Whether Glean should delay persistence of data from metrics with ping lifetime.
```

Reply:
> What makes you think this is not working?

Just my lack of confidence in my understanding of this part of the code, heh.

Reply:
Ok, to improve my confidence I made a little test where I record a metric with a `Lifetime::User` and assert that it `is_some()`. That worked and now I feel better about this test :) Thanks @Dexterp37 for suggesting that.